banner/javascript/sandpack.config.json

```diff
@@ -0,0 +1,13 @@
+{
+  "files": [
+    "/index.html",
+    "/src/styles.css",
+    { "title": "/src/index.js", "active": true }
+  ],
+  "customSetup": {
```

Comment:
what is custom about the setup in our custom sandpack setup?

Reply:
yeah its a sandpack thing